### ISSUE TITLE
About loss_ts, loss_corr

### ISSUE THREAD

**Issue by MIYU8305** [NVlabs/DiscoBox#12]
I did 
bash tools/dist_train.sh \
     configs/discobox/discobox_solov2_r50_fpn_3x.py 2

I got message as below
2022-09-19 00:46:25,659 - mmdet - INFO - Epoch [1][50/29317]    lr: 3.425e-04, eta: 35 days, 8:29:59, time: 2.894, data_time: 1.802, memory: 3714, loss_ins: 1.1359, loss_ts: 0.0000, loss_cate: 0.9793, loss_corr: 0.0000, loss: 2.1152, grad_norm: 11.1556
2022-09-19 00:47:05,600 - mmdet - INFO - Epoch [1][100/29317]   lr: 5.900e-04, eta: 22 days, 13:17:57, time: 0.799, data_time: 0.058, memory: 3714, loss_ins: 1.1685, loss_ts: 0.0000, loss_cate: 0.8634, loss_corr: 0.0000, loss: 2.0319, grad_norm: 9.6288
2022-09-19 00:47:46,070 - mmdet - INFO - Epoch [1][150/29317]   lr: 8.376e-04, eta: 18 days, 7:56:31, time: 0.809, data_time: 0.049, memory: 3714, loss_ins: 1.0841, loss_ts: 0.0000, loss_cate: 0.8087, loss_corr: 0.0000, loss: 1.8928, grad_norm: 9.4327
2022-09-19 00:48:24,677 - mmdet - INFO - Epoch [1][200/29317]   lr: 1.085e-03, eta: 16 days, 2:30:51, time: 0.772, data_time: 0.043, memory: 3714, loss_ins: 1.1373, loss_ts: 0.0000, loss_cate: 0.8160, loss_corr: 0.0000, loss: 1.9532, grad_norm: 8.7715

loss_ts, loss_corr are 0.

It seems it is problem, I can not get any loss from loss_ts, loss_corr. What should I do?

**Comment by voidrank** [comment]
Hi @MIYU8305 ,

The loss_ts, loss_corr won't be turned on util the warmup is finished. FYI, I suggest that you remove loss_corr part in the config file for the new dataset. In that case, you might need less training time, and less hyper-parameter tuning work.

Best,

Shiyi
